x/profiles/types/msgs_dtag_requests_test.go

```diff
@@ -179,13 +179,13 @@ func TestMsgRejectDTagRequest_Route(t *testing.T) {
 }
 
 func TestMsgRejectDTagRequest_Type(t *testing.T) {
```

Comment:
The name of the message is changed to `MsgRefuseDTagTransfer` already. It should follow the name.